src/main/java/me/tongfei/progressbar/ProgressBar.java

```diff
@@ -123,7 +123,7 @@ public ProgressBar(
     @Deprecated
     public ProgressBar start() {
```

Comment:
Honestly I don't see point in keeping this deprecated method in the public API. There is no way anymore (even before this PR) to construct the _ProgressBar_ without the Runnable being started. Personally I would remove it and made the _updateInterval_ private again in _ProgressThread_.